conda/recipes/cudf-polars/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split('.'))[:2] | join('.') }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: cudf-polars
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
```

Comment:
Where is `py_version` coming from? Also, I noticed that RMM's new packages are using `py3.12` instead of `py312`. Can we drop the period?

Reply:
I opened https://github.com/prefix-dev/rattler-build/issues/1449.